src/Feature/Vim/Feature_Vim.re

```diff
@@ -92,6 +92,10 @@ module CommandLine = {
   let%test "meet with a path, spaces" = {
     getCompletionMeet("vsp /path with spaces/") == Some(4);
   };
```

Comment:
Looking at CI - it seems like this test case is failing now. Must be picking up the wrong space - I think your proposal of escaping spaces, and then reversing the logic to find the first non-escaped space, should get both this test and your new one green.

Reply:
yup actually just started working on it, should have a something soon once I figure out how to do stuff resonably in ocaml

Reply:
Cool! We have some utilities that here might be helpful / give some ideas: https://github.com/onivim/oni2/blob/master/src/Core/Utility/StringEx.re

Also the OCaml [`String`](https://caml.inria.fr/pub/docs/manual-ocaml/libref/String.html) and [`Str`](https://caml.inria.fr/pub/docs/manual-ocaml/libref/String.html) might be useful. 

> should have a something soon once I figure out how to do stuff reasonably in ocaml

Seems like you're picking it up fast :smile: